index.html

```diff
@@ -27,7 +27,6 @@
       <div class="logo">
         <img class="img-logo" src="images/logo.svg" alt="logo">
```

Comment:
:red_circle: For semantics reasons, I think your logo image should be wrapped in an `a` tag. Wrapping the image in a tag makes sense for a few reasons. First, it allows users to easily navigate to the homepage of a website by clicking on the logo. Second, it can help to improve brand awareness and recognition. Third, it can improve SEO (search engine optimization).